fs/config/configflags/configflags.go

```diff
@@ -110,6 +110,7 @@ func AddFlags(flagSet *pflag.FlagSet) {
 	flags.StringVarP(flagSet, &fs.Config.StatsOneLineDateFormat, "stats-one-line-date-format", "", fs.Config.StatsOneLineDateFormat, "Enables --stats-one-line-date and uses custom formatted date. Enclose date string in double quotes (\"). See https://golang.org/pkg/time/#Time.Format")
 	flags.BoolVarP(flagSet, &fs.Config.ErrorOnNoTransfer, "error-on-no-transfer", "", fs.Config.ErrorOnNoTransfer, "Sets exit code 9 if no files are transferred, useful in scripts")
 	flags.BoolVarP(flagSet, &fs.Config.Progress, "progress", "P", fs.Config.Progress, "Show progress during transfer.")
+	flags.BoolVarP(flagSet, &fs.Config.ProgressTitle, "print-progress-title", "", fs.Config.ProgressTitle, "Show progress on the terminal title. Requires -P/--progress.")
```

Comment:
Can we call this option `--progress-title` rather than `--print-progress-title` - then it will sort with `--progress` and be easier to discover?

Reply:
What about `--progress-terminal-title`? I think `--progress-title` would not be descriptive enough.